### ISSUE TITLE
Server crash from enderman spawn egg recipe 

### ISSUE THREAD

**Issue by Shekrat** [AllTheMods/ATM-6#338]
**Describe the bug**
The Enderman spawn egg recipe added by Reliquary is causing server crash, was not able reproduce in single creative world.
After the crash the AE2 crafting terminal keeps the content so unless you break it it will keep crashing the server on click.
It is neither craftable in crafting table. It is on 1.3.1 version

**To Reproduce**
1. Go to Crafting terminal
2. Click on the Enderman spawn egg recipe
3. wait for crash

**Expected behavior**
Be able to craft the Enderman spawn egg from the Reliquary recipe

**Screenshots**
![image](https://user-images.githubusercontent.com/32161380/99655143-9b403900-2a5b-11eb-9fd7-e8626a376787.png)

**Additional context**
---- Minecraft Crash Report ----
// I just don't know what went wrong :(

Time: 11/19/20 11:30 AM
Description: Exception in server tick loop

```
[11:30:17] [Server thread/ERROR] [minecraft/MinecraftServer]: Encountered an unexpected exception
java.lang.NoSuchMethodError: net.minecraft.item.SpawnEggItem.func_200889_b(Lnet/minecraft/entity/EntityType;)Lnet/minecraft/item/SpawnEggItem;
        at xreliquary.crafting.FragmentRecipeHelper.getSpawnEggStack(FragmentRecipeHelper.java:51) ~[xreliquary:1.16.3-1.3.4.1055] {re:classloading}
        at java.util.Optional.map(Optional.java:215) ~[?:1.8.0_231] {}
        at xreliquary.crafting.FragmentToSpawnEggRecipe.func_77572_b(FragmentToSpawnEggRecipe.java:39) ~[xreliquary:1.16.3-1.3.4.1055] {re:classloading}
        at xreliquary.crafting.FragmentToSpawnEggRecipe.func_77572_b(FragmentToSpawnEggRecipe.java:19) ~[xreliquary:1.16.3-1.3.4.1055] {re:classloading}
        at appeng.container.implementations.CraftingTermContainer.func_75130_a(CraftingTermContainer.java:111) ~[appliedenergistics2:8.1.0] {re:classloading}
        at appeng.container.implementations.CraftingTermContainer.<init>(CraftingTermContainer.java:87) ~[appliedenergistics2:8.1.0] {re:classloading}
        at appeng.container.implementations.ContainerHelper.lambda$open$2(ContainerHelper.java:116) ~[appliedenergistics2:8.1.0] {re:classloading}
        at net.minecraft.inventory.container.SimpleNamedContainerProvider.createMenu(SourceFile:25) ~[?:?] {re:classloading}
        at net.minecraftforge.fml.network.NetworkHooks.openGui(NetworkHooks.java:222) ~[forge:?] {re:classloading}
        at appeng.container.implementations.ContainerHelper.open(ContainerHelper.java:122) ~[appliedenergistics2:8.1.0] {re:classloading}
        at appeng.container.implementations.ContainerHelper.open(ContainerHelper.java:92) ~[appliedenergistics2:8.1.0] {re:classloading}
        at appeng.container.implementations.CraftingTermContainer.open(CraftingTermContainer.java:60) ~[appliedenergistics2:8.1.0] {re:classloading}
        at appeng.container.ContainerOpener.openContainer(ContainerOpener.java:32) ~[appliedenergistics2:8.1.0] {re:classloading}
        at appeng.parts.reporting.AbstractTerminalPart.onPartActivate(AbstractTerminalPart.java:110) ~[appliedenergistics2:8.1.0] {re:classloading}
        at appeng.parts.AEBasePart.onActivate(AEBasePart.java:409) ~[appliedenergistics2:8.1.0] {re:classloading,pl:runtimedistcleaner:A}
        at appeng.parts.CableBusContainer.activate(CableBusContainer.java:660) ~[appliedenergistics2:8.1.0] {re:classloading}
        at appeng.block.networking.CableBusBlock.onActivated(CableBusBlock.java:341) ~[appliedenergistics2:8.1.0] {re:classloading,pl:runtimedistcleaner:A}
        at appeng.block.AEBaseTileBlock.func_225533_a_(AEBaseTileBlock.java:248) ~[appliedenergistics2:8.1.0] {re:classloading}
        at net.minecraft.block.AbstractBlock$AbstractBlockState.func_227031_a_(AbstractBlock.java:638) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs.mixins.json:AbstractBlockStateMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.server.management.PlayerInteractionManager.func_219441_a(PlayerInteractionManager.java:338) ~[?:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs.mixins.json:PlayerInteractionManagerMixin,pl:mixin:A}
        at net.minecraft.network.play.ServerPlayNetHandler.func_184337_a(ServerPlayNetHandler.java:958) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:fml:randompatches:RandomPatches ServerPlayNetHandler Transformer}
        at net.minecraft.network.play.client.CPlayerTryUseItemOnBlockPacket.func_148833_a(SourceFile:36) ~[?:?] {re:classloading}
        at net.minecraft.network.play.client.CPlayerTryUseItemOnBlockPacket.func_148833_a(SourceFile:10) ~[?:?] {re:classloading}
        at net.minecraft.network.PacketThreadUtil.func_225383_a(SourceFile:21) ~[?:?] {re:classloading}
        at net.minecraft.util.concurrent.TickDelayedTask.run(SourceFile:18) ~[?:?] {re:classloading}
        at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(SourceFile:144) ~[?:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
        at net.minecraft.util.concurrent.RecursiveEventLoop.func_213166_h(SourceFile:23) ~[?:?] {re:mixin,re:computing_frames,re:classloading}
        at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:733) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:jaopca.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:159) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:jaopca.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(SourceFile:118) ~[?:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
        at net.minecraft.server.MinecraftServer.func_213205_aW(MinecraftServer.java:716) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:jaopca.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.server.MinecraftServer.func_213168_p(MinecraftServer.java:710) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:jaopca.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213160_bf(SourceFile:103) ~[?:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
        at net.minecraft.server.MinecraftServer.func_213202_o(MinecraftServer.java:695) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:jaopca.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:645) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:jaopca.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at net.minecraft.server.MinecraftServer.lambda$startServer$0(MinecraftServer.java:232) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:APP:jaopca.mixins.json:MinecraftServerMixin,pl:mixin:A,pl:runtimedistcleaner:A}
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_231] {}
[11:30:17] [Server thread/FATAL] [ne.mi.co.ForgeMod/]: Preparing crash report with UUID 315cd7d6-8192-4ea3-a184-1ff81a4f540a

**Comment by ViRb3** [comment]
This looks to be an issue with [Reliquary](https://www.curseforge.com/minecraft/mc-mods/reliquary-v1-3). A similar issue has been raised on their issue tracker:
https://github.com/P3pp3rF1y/Reliquary/issues/478

**Comment by ViRb3** [comment]
This issue has now been fixed upstream, expect it to land in ATM6 on next update.

**Comment by ViRb3** [comment]
Fixed in ATM6 1.3.2b
